src/card.css

```diff
@@ -0,0 +1,86 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  align-self: center;
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  align-items: center;
+  font-weight: 500;
+  line-height: 18px;
+  font-size: 12px;
+  color: #060b35;
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
+  color: #616070;
+  margin-top: 4px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 10px;
+  color: #060b35;
+  margin-top: 17px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 23px;
+  margin-bottom: 16px;
+}
+
+.card__price-title {
+  color: #616070;
+}
+
+.card__price-number {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+}
+
+.card__button {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+
+  width: 166px;
+  height: 40px;
+  font-size: 14px;
+  font-weight: 700;
+
+  text-decoration: none;
+  text-transform: uppercase;
+
+  color: white;
+  background: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+}
+
+.card__button:hover {
+  width: 166px;
+  height: 40px;
```

Comment:
you need to add ob hover styles that will change but width and height won’t charge (default width and height the same)